pkg/collector/corechecks/embed/process_agent.go

```diff
@@ -129,7 +129,8 @@ func (c *ProcessAgentCheck) run() error {
 func (c *ProcessAgentCheck) Configure(data integration.Data, initConfig integration.Data, source string) error {
 	// handle the case when the agent is disabled via the old `datadog.conf` file
 	if enabled := config.Datadog.GetBool("process_config.enabled"); !enabled {
```

Comment:
Will it be a problem that things like `source/binPath` won't be set for calls to `run()` later?

Reply:
You are right, I removed the early return. Thanks.